Add entry - HTML: Tags: `<kbd>` (#616)
Codecademy/docs: content/html/concepts/tags/terms/kbd/kbd.md

---
Title: '<kbd>'
Description: 'Makes charecters look like keys on a keyboard'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<kbd>` tag makes characters look like they are keys on a keyboard.

## Syntax

> All that is needed to be done is put the characters between an opening and closing tag. The `<kbd>` tag has no attributes. They can be nested but the result will be the same.

Reviewer: Should have a pseudo or html block with something like `<kbd><!-- text goes here --></kbd>

Reviewer 2: `<kbd><!-- Text goes here --></kbd>`**

## Example 1

```html

> <p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>

Reviewer: Applies to this and below. Generally, when referring to alpha keys on the keyboard, they're usually capitalized.

```

Result:

> You can copy text by pressing ``CTRL`` + ``c``.

Reviewer: This is a case where we could use an image of the rendered text. The picture can be placed in the /media folder.

## Example 2

```html
<p>You can peaste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>v</kbd></kbd>.</p>
```
Result:

> You can peaste text by pressing ``CTRL`` + ``c``.

Reviewer: ```suggestion
You can paste text by pressing ``CTRL`` + ``v``.
```

---
Title: '<kbd>'
Description: 'Makes charecters look like keys on a keyboard'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<kbd>` tag makes characters look like they are keys on a keyboard.

## Syntax

> All that is needed to be done is put the characters between an opening and closing tag. The `<kbd>` tag has no attributes. They can be nested but the result will be the same.

Reviewer 3: ```suggestion
The `<kbd>` tag has no attributes. Text is placed in between the opening and closing `<kbd>` tags and will render in the browser's default monospace font. `<kbd>` can be nested but the result will be the same.
```

```pseudo
<kbd><!-- Text goes here --></kbd>
```
  
## Example 1

```html

> <p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>

Reviewer 3: ```suggestion
<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>C</kbd>.</p>
```

```

Result:

> ![picture of the <kbd> in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb%20tag%20example%201.png)

Reviewer 3: The file names for the images need to be changed to something like "kbd-tag-image.png" where  "-" is used for spacing. The image link will look something like this "https://raw.githubusercontent.com/Codecademy/docs/main/media/kbd-tag-image.png".  Take a look in the media folder to see how the names are written. Or you can take a look at the [recently merged  HTML `<span>` entry](https://github.com/Codecademy/docs/pull/605/files) which uses an image.

  
## Example 2

```html

> <p>You can peaste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>v</kbd></kbd>.</p>

Reviewer 3: ```suggestion
<p>You can paste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>V</kbd></kbd>.</p>
```

---
Title: '<kbd>'
Description: 'Emphasizes characters to look like keys on a keyboard.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Style'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<kbd>` tag emphasizes characters to look like keys on a keyboard.

## Syntax

The `<kbd>` tag has no attributes. Text is placed in between the opening and closing `<kbd>` tags and will render in the browser's default monospace font. `<kbd>` can be nested but the result will be the same.

```pseudo
<kbd><!-- Text goes here --></kbd>
```
  
## Example 1

```html
<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>C</kbd>.</p>
```

Result:

> ![Keyboard Tag Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-1.png)

Reviewer 4: I noticed a small typo with the filenames. It should say "*kbd*-tag-example-1.png" (emphasis mine), instead. We will need to rename both this file and the other, too.

To make this change, you have to make it locally by running the following command:

```
git mv kdb-tag-example-1.png kbd-tag-example-1.png
```

Then, run `git add` + `commit` + `push` back up to this branch.

> ![picture of the <kbd> in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb-tag-example-1.png)

Reviewer 2: ```suggestion
![Keyboard Tag Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-1.png)
```

  
## Example 2

```html
<p>You can paste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>V</kbd></kbd>.</p>
```
  
Result:

> ![Keyboard Tag Example 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-2.png)

Reviewer 4: We need to make a similar local change here by running the following command:

```
git mv kdb-tag-example-2.png kbd-tag-example-2.png
```

Then, run `git add` + `commit` + `push` back up to this branch.

> ![picture of the <kbd> tag in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb-tag-example-2.png)

Reviewer 2: ```suggestion
![Keyboard Tag Example 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-2.png)
```
